### PR TITLE
PARQUET-342: Updates to be Java 6 compatible

### PR DESCRIPTION
**Pull request by nezihyigitbasi**

### CODE REVIEW

File: parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestInputOutputFormatWithPadding.java [hunk 1]
```diff
@@ -190,6 +189,23 @@ public void testBasicBehaviorWithPadding() throws Exception {
     ParquetFileWriter.BLOCK_FS_SCHEMES.remove("file");
   }
 
+  private List<String> readAllLines(File file) throws IOException {
```
Comment (isnotinvain) [inline parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestInputOutputFormatWithPadding.java]: maybe we can put this in a shared file so it's not duplicated, maybe in org.apache.parquet.Files, next to org.apache.parquet.Strings?

Comment (nezihyigitbasi) [inline parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestInputOutputFormatWithPadding.java]: @isnotinvain Yep makes sense. Updated.

**Comment by nezihyigitbasi** [comment]
@isnotinvain let me know if you have any other comments.

File: parquet-common/src/main/java/org/apache/parquet/Files.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+//TODO: Use java.nio.file.Files when Parquet is updated to Java 7
+public final class Files {
+  private Files() { }
+
+  public static List<String> readAllLines(File file) throws IOException {
```
Comment (isnotinvain) [inline parquet-common/src/main/java/org/apache/parquet/Files.java]: this should accept a charset as an argument and thread it through

**Comment by isnotinvain** [comment]
Sorry, just one last thing but +1 other than that

**Comment by nezihyigitbasi** [comment]
@isnotinvain yep makes sense, updated.

**Comment by isnotinvain** [comment]
+1 thanks!

**Pull request closed without merging**